tests/page/page-autowaiting-basic.spec.ts

```diff
@@ -15,6 +15,7 @@
  * limitations under the License.
  */
 
+import { stripAnsi } from 'tests/config/utils';
```

Comment:
nit/realisation: In our code-base we have an inconsistent situation across imports. `../../packages/playwright` vs `packages/playwright`. Ditto with `tests/config`.

Reply:
landing as a follow-up